orcestra/flightplan.py

```diff
@@ -658,34 +772,142 @@ def to_geojson(path):
     )
 
 
+def az_to_text(az):
+    sections = [
+        "north",
+        "north east",
+        "east",
+        "south east",
+        "south",
+        "south west",
+        "west",
+        "north west",
+    ]
+    return sections[int(((az + (360 / 2 / len(sections))) * len(sections) / 360) % 8)]
+
+
+def to_detailed_txt(plan: FlightPlan):
+    from io import StringIO
```

Comment:
I would argue that it is fine to import packages from the standard library at the top of the module, but I am fine with keeping it lazy

Reply:
My idea was, that we probably want to get rid of StringIO (it's only there to be close to the original). Thus my thinking was to keep it local.

Reply:
I'm fine with that